users/config.rst

```diff
@@ -341,6 +349,99 @@ markerName
     Name of a directory or file in the folder root to be used as
     :ref:`marker-faq`. Default is ".stfolder".
 
+copyOwnershipFromParent
+    On Unix systems, tries to copy file/folder ownership from the parent directory (the directory it's located in).
+    Requires running Syncthing as privileged user, or granting it additional capabilities (e.g. CAP_CHOWN on Linux).
+
+modTimeWindowS
+    Allowed modification timestamp difference when comparing local and remote file for equivalence.
+    To be used on systems that have unstable modification timestamps, that might change after being observed after
+    the last write operation. Used in Android only.
+
+maxConcurrentWrites
+    Maximum number of concurrent write operations while syncing. Defaults to 2. Increasing this might increase or
+    decrease disk performance, depending on the underlying storage.
+
+disableFsync
+    Disables committing file operations to disk before recording them in the database.
```

Comment:
I'd discourage usage even more, something like this:

```suggestion
    Do not enable this option unless you know exactly what you are doing.
    Disables committing file operations to disk before recording them in the database.
```

And or use the "warning-thingy" (just discovered below).